### PR TITLE
[FIX]: Docker Configs

### LINKED ISSUE #251: [FIX]: Docker Configs in Dev Mode
## Description

**Docker** configurations not working properly in development mode

### Steps to reproduce

1. Follow the [how to run](https://github.com/xgeekshq/split/wiki/How-to-run#development-mode) steps;
2. Open frontend on yout browser (`http://localhost:3000`) and try login
3. Appears a connection error

### Expected result

Request's between frontend and backend working properly, without any connection error message.

### PR DESCRIPTION
**Pull request by dsousa12**
Fixes #251 

## Proposed Changes

  - Change the `.env` variables
  - Update `docker-compose` and `Dockerfile` (backend) file

This pull request closes #251

### CODE REVIEW

**Comment by CatiaBarroco-xgeeks** [comment]
Already tested , and it's working

**Comment by joaosantos99** [comment]
I tested and work's, without a problem.

**Comment by geomarb** [comment]
It is working fine now! Thanks Daniel Sousa!!

**Review by nunocaseiro — APPROVED** [review]

File: frontend/src/hooks/useUser.tsx [hunk 1]
```diff
@@ -13,12 +11,9 @@ import {
 	ResetTokenResponse,
 	UseUserType
 } from 'types/user/user';
-import { transformLoginErrorCodes } from 'utils/errorCodes';
 import { DASHBOARD_ROUTE } from 'utils/routes';
 
-const useUser = (setErrorCode?: Dispatch<SetStateAction<number>>): UseUserType => {
```
Comment (dsousa12) [inline frontend/src/hooks/useUser.tsx:19]: @nunocaseiro **ESLint** show me an error on `setErrorCode` parameter on each commit (`error  'setErrorCode' is defined but never used`) so I removed the parameter and the `const router = useRouter()`.

**Pull request merged**